ibis/backends/tests/test_generic.py

```diff
@@ -2125,22 +2125,7 @@ def test_dynamic_table_slice_with_computed_offset(backend):
 
 
 @pytest.mark.notimpl(["druid", "risingwave"], raises=com.OperationNotDefinedError)
-@pytest.mark.parametrize(
-    "method",
-    [
-        "row",
-        param(
-            "block",
-            marks=[
-                pytest.mark.notimpl(
-                    ["snowflake"],
-                    raises=SnowflakeProgrammingError,
-                    reason="SAMPLE clause on views only supports row wise sampling without seed.",
```

Comment:
This error message was a bit confusing to me. I initially thought it was saying "SAMPLE on subqueries with no seed only works row wise" (implying that if a seed was set, then block wise would work), but actualy it means "SAMPLE on subqueries only works row wise with no seed"